charts/devtron/templates/helm-client.yaml

```diff
@@ -0,0 +1,92 @@
+{{- if or (eq .Values.installer.mode "hyperion") (eq .Values.installer.mode "Hyperion") }}
+{{- with .Values.hyperion.helmClient }}
+apiVersion: v1
```

Comment:
we are replacing `helm-client` with  `symbiote`

Reply:
it was due to missing commit in previous push where helm-client.yaml was deleted, but it's done now